phpBB/includes/functions_jabber.php

```diff
@@ -87,8 +87,7 @@ function jabber($server, $port, $username, $password, $use_ssl = false)
 	*/
 	static public function can_use_ssl()
 	{
-		// Will not work with PHP >= 5.2.1 or < 5.2.3RC2 until timeout problem with ssl hasn't been fixed (http://bugs.php.net/41236)
-		return ((version_compare(PHP_VERSION, '5.2.1', '<') || version_compare(PHP_VERSION, '5.2.3RC2', '>=')) && @extension_loaded('openssl')) ? true : false;
+		return @extension_loaded('openssl') ? true : false;
```

Comment:
`extension_loaded` returns a `bool`, so I will simply change it to `return @extension_loaded('openssl');`.